astroid/modutils.py

```diff
@@ -33,22 +33,24 @@
 :type BUILTIN_MODULES: dict
 :var BUILTIN_MODULES: dictionary with builtin module names has key
 """
+
+# We don't want distutils to be a requirement
```

Comment:
```suggestion
# TODO: We don't want distutils to be a requirement
```
?

Reply:
(We disable the import-error so pylint can work without distutil installed.) I updated the comment :)